### DISCUSSION TITLE
New button "Operator arguments"? between "Task Instance Details" and "Rendered"?

### DISCUSSION THREAD

**Discussion by JavierLopezT** [apache/airflow#17267] [Ideas] [answered]
Hello. A lot of times, when I click on a task, I just want to see the operator arguments that can be manually passed, so Rendered view lacks information, because there are not non-renderable arguments, and Task Instance Details has way too much information, because I don't care about most of the things that are there.

I think it would be useful to add a feature to accomplish this. One way could be a new button/view called Task Operator arguments or something like that, or maybe Rendered could have all the manual arguments. 

What do you think?

**Comment by potiuk** [comment, marked as answer]
Feel free to open PR for that :)

**Reply by JavierLopezT** [reply]
Cool, thanks. What do you think would be a good name for it?
